src/components/CollapsibleWrap/CollapsibleWrap.tsx

```diff
@@ -0,0 +1,29 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { AnimatePresence, motion } from "framer-motion";
+import React, { FC } from "react";
+import { CollapsibleWrapProps } from "./types";
+
+export const CollapsibleWrap: FC<CollapsibleWrapProps> = ({
+    children,
+    isOpen = false,
+    preventInitialAnimation = false,
+}) => (
+    <AnimatePresence initial={preventInitialAnimation ? false : undefined}>
+        {isOpen && children && (
+            <motion.div
+                initial={"collapsed"}
+                animate={"open"}
+                exit={"collapsed"}
+                variants={{
+                    open: { height: "auto", overflow: "hidden" },
+                    collapsed: { height: 0, overflow: "hidden" },
+                }}
+                transition={{ type: "tween" }}
```

Comment:
![Screenshot 2022-03-23 at 11 17 28](https://user-images.githubusercontent.com/3276065/159676842-6296b7f6-36b6-43ba-8561-c20c55dea205.png)
Setting overflow hidden prevents dropdown from going out of an accordion item.

Reply:
Intended behavior, the dropdown should be always on top, so the fix should be there (not only an issue in accordion)

Reply:
@jonasbuechel I could revert this, however a problem can occur in that while the accordion is expanding there will be a visual glitch where overflowing content (which is all content at the beginning since the height is 0) will briefly be displayed over the Accordion below until the Accordion reaches its full size